content/en/docs/concepts/architecture/cgroups.md

```diff
@@ -0,0 +1,128 @@
+---
+title: About cgroup v2
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+
+On Linux, {{< glossary_tooltip text="control groups" term_id="cgroup" >}}
+constrain resources that are allocated to processes.
+
+The {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} and the
+underlying container runtime need to interface with cgroups to enforce
+[resource mangement for pods and
+containers](/docs/concepts/configuration/manage-resources-containers/) which
+includes cpu/memory requests and limits for containerized workloads.
+
+There are two versions of cgroups in Linux: cgroupv1 and cgroup v2. cgroup v2 is
+the new generation of the `cgroup` API.
+
+<!-- body -->
+
+
+## What is cgroup v2? {#cgroup-v2}
+{{< feature-state for_k8s_version="v1.25" state="stable" >}}
+
+Cgroup v2 is the next version of the Linux `cgroup` API. cgroup v2 provides a
+unified control system with enhanced resource management
+capabilities.
+
+cgroup v2 offers several improvements over cgroup v1, such as the following:
+
+- Single unified hierarchy design in API
+- Safer sub-tree delegation to containers
+- Newer features like [Pressure Stall Information](https://www.kernel.org/doc/html/latest/accounting/psi.html)
+- Enhanced Resource allocation management and isolation across multiple resources
+  - Unified accounting for different types of memory allocations (network memory, kernel memory, etc)
+  - Accounting for non-immediate resource changes such as page cache write backs
```

Comment:
```suggestion
  - Accountability for non-immediate resource changes such as page cache write backs
```
If you disagree and think that accounting makes more sense, feel free to ignore

Reply:
I think accounting is the term usually used to describe this, e.g. from the man pages:

>  At each level of
       the hierarchy, attributes (e.g., limits) can be defined.  The
       limits, control, and **accounting** provided by cgroups generally
       have effect throughout the subhierarchy underneath the cgroup
       where the attributes are defined. 

Will leave it as is to match that.

Reply:
SGTM